src/database/video_emotion_database.py

```diff
@@ -26,11 +26,12 @@ def __init__(self):
             username=cfg['mongo_user'], password=cfg['mongo_pass'])
         self._db = self._client[cfg['mongo_db']]
 
-    def insert(self, col,
-               song_id, video_id, doc):
-        collection = self._db[col]
+    def insert(self, name,
+               song_id, video_id, time, emotion):
+        collection = self._db[name]
```

Comment:
It should really use "col" instead of "name". As it isn't the name of the database, it's the collection name of the database.